core/tee/tee_fs_key_manager.c

```diff
@@ -68,13 +69,19 @@ static struct tee_fs_ssk tee_fs_ssk;
 static uint8_t string_for_ssk_gen[] = "ONLY_FOR_tee_fs_ssk";
 
 
+static TEE_Result do_hmac(uint8_t *out_key, uint32_t out_key_size,
+			  const uint8_t *in_key, uint32_t in_key_size,
+			  const uint8_t *message, uint32_t message_size);
```

Comment:
Kernel-style would say to just move this function up so the static declaration isn't needed. It doesn't particularly bother me, however.